services/datamanager/builtin/sync_test.go

```diff
@@ -384,6 +385,7 @@ func TestArbitraryFileUpload(t *testing.T) {
 			cfg.ScheduledSyncDisabled = tc.scheduleSyncDisabled
 			cfg.SyncIntervalMins = syncIntervalMins
 			cfg.AdditionalSyncPaths = []string{additionalPathsDir}
+			cfg.AdditionalTags = [][]string{tags}
```

Comment:
Are there any tests that confirm this works with different tags per path?

Reply:
I tested this manually but can add a unit test for it!